app/controllers/application_controller.rb

```diff
@@ -1,10 +1,22 @@
 require './config/environment'
+require 'pry'
 
 class ApplicationController < Sinatra::Base
 
   configure do
+    enable :sessions unless test?
+    set :session_secret, "secret"
+  end
+
+  include Helpers
```

Comment:
This is not a correct pattern. Sinatra provides ApplicationController level helpers through a [`helpers`](http://www.sinatrarb.com/intro.html#Helpers) macro. You would not need to call the methods defined in the `helpers` block with a module prefix as you do now `Helpers.logged_in?`.